.circleci/config.yml

```diff
@@ -1,23 +1,40 @@
-version: 2
-jobs:
+version: 2.1
+
+jobs: 
   build:
     docker:
-      - image: circleci/node:10
+      - image: circleci/node
+
+    parameters:
+      eslint:
+        type: string
 
     steps:
       - checkout
 
       - restore_cache:
           keys:
-          - v1-dependencies-{{ checksum "yarn.lock" }}
+          - v1-dependencies-{{ checksum "yarn.lock" }}-<< parameters.eslint >>
           - v1-dependencies-
```

Comment:
What is this cache key used for?

Reply:
We make CircleCI cache the NPM dependencies between builds so we don't have to redownload them. 
Based on your other comments, I will rework the CI config.